cypress/integration/userCanCreateTimeRecording.feature.js

```diff
@@ -4,43 +4,50 @@ describe("User can create time recording", () => {
 	it("successfully", () => {
 		cy.server();
 		cy.route({
-			method: 'POST',
-			url: 'https://demo-stable.kimai.org/api/timesheets',
-			response: 'fixtures:save_data.json',
+			method: "POST",
+			url: "https://demo-stable.kimai.org/api/timesheets",
+			response: "fixtures:save_data.json",
 			headers: {
 				"X-AUTH-USER": "susan_super",
 				"X-AUTH-TOKEN": "api_kitten"
 			}
-		})
-		cy.visit("http://localhost:3000")
-		cy.get('input[type=name]').type('anna_admin')
-		cy.get('input[type=password]').type('api_kitten')
-		cy.get('button[type=submit]').click()
-		cy.get('input[name="begin"]').type("07:00")
-		cy.get('input[name="end"]').type("08:00")
-		cy.get('button[name="create"]').click()
-		cy.contains('Your time was saved')
-	})
+		});
+		cy.visit("http://localhost:3000");
+		cy.get("input[type=name]").type("anna_admin");
+		cy.get("input[type=password]").type("api_kitten");
+		cy.get("button[type=submit]").click();
+		cy.get("#customer > .dropdown").click();
+		cy.contains("Hammes-Kilback").click();
+		cy.get("#projects > .dropdown").click();
+		cy.contains("Advanced content-based functionalities").click();
+		cy.get('input[name="begin"]').type("07:00");
+		cy.get('input[name="end"]').type("08:00");
+		cy.get(':nth-child(9) > input').type("25");
+		cy.get('button[name="create"]').click();
+		cy.contains("Your time was saved");
```

Comment:
I would suggest this standard for assertions. 
```
cy.get('#id_of_element').should('contain', 'Your time was saved')
```

Reply:
The tests complain about async and the button is not attached to the dom when using this assertion..

Reply:
```CypressError: Timed out retrying: cy.should() failed because this element is detached from the DOM. ```

Reply:
Fixed , added the new syntax. I was missing Id's for the p tag that held the error/success messages